nexus/src/app/instance.rs

```diff
@@ -1831,14 +1830,22 @@ impl super::Nexus {
     }
 }
 
-/// Invoked by a sled agent to publish an updated runtime state for an
-/// Instance, returning an update saga for that instance (if one must be
-/// executed).
+/// Publishes the VMM and migration state supplied in `new_runtime_state` to the
```

Comment:
super nitpicky, but i'm not sure if "publish"  is my favorite word for updating a database record...I can't think of an obvious alternative OTTOMH though.

Reply:
I dug deep on this one and came up with "write" :) Fixed in fb9ee7e.